src/components/AccountDetails/index.tsx

```diff
@@ -3,10 +3,11 @@ import { useWeb3React } from '@web3-react/core'
 import CopyHelper from 'components/AccountDetails/Copy'
 import { coinbaseWalletConnection, injectedConnection } from 'connection'
 import { getConnection } from 'connection/utils'
-import { useCallback, useContext } from 'react'
+import React, { useCallback, useContext } from 'react'
```

Comment:
Import `Context` directly to avoid importing all of `React`.

Reply:
done